build.gradle

```diff
@@ -44,6 +41,11 @@ compileGroovy {
     targetCompatibility '1.7'
 }
 
+tasks.withType(ValidatePlugins).configureEach {
+    // TODO remove once these issues are fixed
```

Comment:
what issues is this ignoring? and how can we get them fixed?

Reply:
All properties in tasks must be annotated with input/output annotations. You'll see the errors when you run `./gradlew build`. I didn't want to touch things because adding input annotations has caching implications and I don't fully understand which tasks you run for a release.